scripts/exhafs_grid.sh

```diff
@@ -172,10 +174,29 @@ elif [ $gtype = regional ]; then
   # number of parent points
   nptsx=`expr $iend_nest - $istart_nest + 1`
   nptsy=`expr $jend_nest - $jstart_nest + 1`
+ # 
+  i_center=`expr $nptsx / 2`
```

Comment:
@BantwaleEnyew-NOAA Could you please revert these changes, which should not be part of this PR.